sourcery_analytics/settings.py

```diff
@@ -1,24 +1,37 @@
+"""Models describing sourcery-analytics settings."""
 import pathlib
 
 import pydantic
 import tomli
 
 
 class ThresholdSettings(pydantic.BaseModel):
+    """Model describing the available thresholds and their defaults."""
```

Comment:
**suggestion (docstrings):** Please update the docstring for function: `ThresholdSettings`

Reason for update: The docstring was added to the class.

Suggested new docstring:
```
"""Model describing the available thresholds and their defaults."""
```